.test-infra/jenkins/dependency_check/dependency_check_report_generator.py

```diff
@@ -176,7 +187,77 @@ def compare_dependency_versions(curr_ver, latest_ver):
   return False
 
 
-def query_dependency_release_dates(bigquery_client, dep_name, curr_ver_in_beam, latest_ver):
+def find_release_time_from_maven_central(group_id, artifact_id, version):
+  """
+  Find release dates from Maven Central REST API.
+  Args:
+    group_id:
+    artifact_id:
+    version:
+  Return:
+    release date
+  """
+  url = "http://search.maven.org/solrsearch/select?q=g:{0}+AND+a:{1}+AND+v:{2}".format(
+      group_id,
+      artifact_id,
+      version)
+  logging.info('Finding release from maven central')
+  try:
+    response = request_session_with_retries().get(url)
+    if not response.ok:
+      logging.error("The response status code is not ok: " + str(response.status_code))
+      logging.info("Failed finding the release date of {0}:{1} {2}".format(group_id, artifact_id, version))
+      return None
+    response_data = response.json()
+    release_timestamp = response_data['response']['docs'][0]['timestamp']
```

Comment:
Are all these fields guaranteed to be available ?

Reply:
Sometimes no. The KeyError will be caught if fields are not exist. Return "None" and use the BigQuery as backup.